src/ontology/pato-edit.obo

```diff
@@ -22136,6 +22136,49 @@ xref: ILX:0770100
 is_a: PATO:0070033 ! neuron projection quality
 created_by: http://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070036
+name: dendrite quality
+comment: A morphological quality inhering in a neuron's dendrites by virtue of the bearer's physical composition.
+xref: ILX:0103022
+is_a: PATO:0000051 ! morphology
```

Comment:
I thought that we were going to make dendrite quality broader than morphology?

Reply:
ah sorry, misunderstood, yeah happy to move it up to quality

Reply:
actually would it make more sense to be asserted as a 'physical object quality'(PATO:0070036)?

Reply:
I think that `physical object quality` is for monadic qualities only. If we want to include non-monadic (relational) qualities at some time, then it should be under `quality`.